go/Dockerfile

```diff
@@ -26,7 +26,7 @@ WORKDIR /root/
 COPY --from=builder /app/main .
 
 # Expose port
-EXPOSE 50051
+EXPOSE 50055
```

Comment:
How did this work? :astonished:

Reply:
I mean... We're not using that port. No? :sweat_smile: 

We're always overriding `PORT`, and we're exposing it on docker-compose.yml/the task definition.

This was just me trying to have less confusing things in the repo.

Reply:
Oh yes, you are right.